staging/publishing/rules.yaml

```diff
@@ -1640,9 +1640,9 @@ rules:
       branch: master
       dir: staging/src/k8s.io/controller-manager
     name: master
-    go: 1.13.9
+    go: 1.13.15
```

Comment:
master would be using go 1.15 so we'd need to remove this line (was incorrect before)

Reply:
Fixed.